src/Query/TermsQuery.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Solr Client Symfony package.
+ *
+ * (c) ingatlan.com Zrt. <fejlesztes@ingatlan.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace iCom\SolrClient\Query;
+
+final class TermsQuery
+{
+    private $params = [
+        'field' => null,
+        'separator' => ',',
+        'method' => null,
+        'values' => null,
+    ];
+
+    public function __construct(string $field, array $values)
+    {
+        $this->params['field'] = $field;
+        $this->params['values'] = $values;
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('{!terms f=%s%s%s}%s',
```

Comment:
`return sprintf('{!terms f=%s}%s', urldecode(http_build_query($this->params, '', ' ')), implode($this->params['separator'] ?? ',', $this->values));`